docs/topics/native/native-objc-interop.md

```diff
@@ -43,11 +43,11 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a framewor
 >
 {style="warning"}
 
-To make your Kotlin code more Objective-C/Swift-friendly, you can hide a Kotlin declaration from Objective-C and Swift
+To make your Kotlin code more Swift/Objective-C-friendly, you can hide a Kotlin declaration from Objective-C and Swift
 with `@HiddenFromObjC`. The annotation disables a function or property export to Objective-C.
 
 Alternatively, you can mark Kotlin declarations with the `internal` modifier to restrict their visibility in the
-compilation module. Choose `@HiddenFromObjC` if you only want to hide the Kotlin declaration from Objective-C and Swift,
+compilation module. Choose `@HiddenFromObjC` if you only want to hide the Kotlin declaration from Objective-C and Swift
```

Comment:
```suggestion
compilation module. Use `@HiddenFromObjC` if you only want to hide the Kotlin declaration from Objective-C and Swift
```

Reply:
Or maybe it is better to rephrase the whole sentence:

Use @HiddenFromObjC if you want to hide the Kotlin declaration from Objective-C and Swift while keeping it visible to other Kotlin modules.